scripts/build-asm.py

```diff
@@ -0,0 +1,212 @@
+#!/usr/bin/env python
```

Comment:
- needs license header
- `sanity.sh` scripts needs to have python file checking